src/typing_extensions.py

```diff
@@ -2496,6 +2609,21 @@ def _is_unpack(obj):
     class _UnpackAlias(typing._GenericAlias, _root=True):
         __class__ = typing.TypeVar
 
+        def _typing_unpacked_tuple_args(self):
+            assert self.__origin__ is Unpack
+            assert len(self.__args__) == 1
+            arg, = self.__args__
+            if isinstance(arg, typing._GenericAlias):
+                if arg.__origin__ is not tuple:
+                    raise TypeError("Unpack[...] must be used with a tuple type")
+                return arg.__args__
+            return None
+
+        def __getattr__(self, attr):
+            if attr == '__typing_unpacked_tuple_args__':
```

Comment:
Why does this need to go through `__getattr__`? Why can't we add a property named `__typing_unpacked_tuple_args__`?

Reply:
I had a reason once, but I do not see it anymore. Is now a property.